GiniVision/Classes/Core/DocumentPickerCoordinator.swift

```diff
@@ -59,19 +111,29 @@ public final class DocumentPickerCoordinator: NSObject {
         }
     }
     
-    // MARK: - Initializer
-    
+    /**
+     Designated initializer for the `DocumentPickerCoordinator`.
+     
+     - parameter giniConfiguration: `GiniConfiguration` use to configure the pickers.
```

Comment:
Didn't you remove the Gini in `giniConfiguration` in some classes? Either have it everywhere or remove it everywhere.

Reply:
It is only `configuration` in the `GiniVision` methods, and since they are a public interface, I don't want to change them.